charts/kargo/templates/api/ingress.yaml

```diff
@@ -16,10 +16,10 @@ spec:
   ingressClassName: {{ .Values.api.ingress.ingressClassName }}
   {{- end }}
   rules:
-  - host: {{ .Values.api.host }}
+  - host: {{ quote .Values.api.host }}
     http:
       paths:
-      - pathType: {{ .Values.api.ingress.pathType }}
+      - pathType: {{ .Values.api.ingress.pathType | default "ImplementationSpecific" }}
```

Comment:
`ImplementationSpecific` is the default in `values.yaml`, but this must never be blank. Default added here to account for the possibility of a user changing it to `""`.